test/test_euler.py

```diff
@@ -750,6 +773,9 @@ def rhs(t, q):
                 write_soln()
 
         fields = rk4_step(fields, t, dt, rhs)
+        fields = filter_modally(discr, "vol", cutoff=cutoff,
+                                       mode_resp_func=frfunc, field=fields)
```

Comment:
It would be interesting to test this someplace that otherwise blows up due to aliasing.

Reply:
Yes!